<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartPage</class>
 <widget class="QDialog" name="StartPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="windowName">
     <property name="text">
      <string>Get Started</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QStackedWidget" name="tablePlace">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_5"/>
       <widget class="QWidget" name="page_6"/>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="select_btn">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Select File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear_btn">
         <property name="text">
          <string>Clear HIstory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="help_btn">
         <property name="text">
          <string>Get Help</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancle_btn">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Cancle</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="without_btn">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>continue without dataBase</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="appname">
     <property name="text">
      <string>School Managment 2025</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
